src/pkgcheck/checks/metadata_xml.py

```diff
@@ -467,6 +514,15 @@ def _check_longdescription(self, pkg, loc, doc):
                 msg = 'metadata.xml longdescription is too short'
                 yield RedundantLongDescription(msg, pkg=pkg)
 
+    def _check_remote_id(self, pkg, loc, doc):
+        for u in pkg.upstreams:
+            try:
+                validator, expected = self.remote_id_validators[u.type]
+            except KeyError:
+                continue
```

Comment:
Do I understand correctly that this applies for when the remote-id type is incorrect?
If yes, should we throw another Error/Warning? Also maybe we should add it into `testdata/repos/standalone/PackageMetadataXmlCheck/InvalidRemoteID`?

Reply:
Remote-id types are checked through XML Schema. This is primarily meant to allow adding new types without having to update the check immediately.